psi4/src/read_options.cc

```diff
@@ -1199,7 +1199,7 @@ int read_options(const std::string &name, Options & options, bool suppress_print
     /*- What algorithm to use for the SCF computation. See Table :ref:`SCF
     Convergence & Algorithm <table:conv_scf>` for default algorithm for
     different calculation types. -*/
```

Comment:
@loriab Where is a good place to add discussion between `DF/MEM_DF/DISK_DF`?

Reply:
somewhere below [here](https://github.com/psi4/psi4/blob/master/doc/sphinxman/source/scf.rst#eri-algorithms) or another section if need be.